website/docs/Ganglion/09-Ganglion_Programming_Tutorial.md

```diff
@@ -2,41 +2,45 @@
 id: GanglionProgram
 title: Ganglion Programming Tutorial
 ---
-**Please note, you do NOT need to program the Ganglion in order to use it. All OpenBCI boards ship ready to use out of the box. This guide is for users who want to upload their own firmware to the Ganglion or modify existing firmware.**
+:::tip
```

Comment:
This probably needs to be `caution`, as that's what I was about to change it to.

Reply:
I would argue that `caution` has a negative connotation to it. I think this statement is positive because users don't have to do the process if they don't want to. They can still program it if they want (so no harm will be done), but they aren't required to.